wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceFSStorageCleaner.java

```diff
@@ -0,0 +1,28 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2016 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.api.workspace.server;
+
+/**
+ * This component removes workspace folder from workspace storage. It's used for "delete workspace" operation
```

Comment:
> It's used for "delete workspace" operation

If this interface exists for certain needs please document this needs better, as it's not clear what the phrase above means. As far as i understand we need this interface for cleaning file system resources after workspace is removed?